.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
   test_macos:
     macos:
-      xcode: "9.0"
+      xcode: "11.3.0"
```

Comment:
Any chance we can do this in a separate PR? Feel free to tag me.
Just want to isolate infra/tooling changes from other things, makes it easier to revert in case we need to.

Reply:
yeap, I think that makes sense. Will do that now